pyprocessing/environment.py

```diff
@@ -5,10 +5,8 @@ def size(w, h):
     global pp
     global width
     global height
-    pp.namespace['width'] = w
-    pp.namespace['height'] = h
-    width = w
-    height = h
```

Comment:
These variables are part of the processing specification

See https://processing.org/reference/width.html and https://processing.org/reference/height.html

Reply:
What if I added `pp.width = w` and `pp.height = h` without removing the other lines? I think it would fix the issue and wouldn't regress the code.